python/Ganga/Utility/Runtime.py

```diff
@@ -236,6 +236,42 @@ def postBootstrapHook(self):
             logger.debug("no postBootstrapHook() in runtime package %s", self.name)
 
 
+def initSetupRuntimePackages():
+    """ Short wrapper function to init and setup Runtime Plugins """
+    initRuntimePackages()
+
+    # perform any setup of runtime packages
+    logger.debug('Setting up Runtime Packages')
+    for r in allRuntimes.values():
+        r.standardSetup()
+
+
+def initRuntimePackages():
+    """
+    initialize runtime packages, they are registered in allRuntimes
+    dictionary automatically
+    """
+    import Ganga.Utility.files
+    from Ganga.Utility.Config.Config import getConfig
+    config = getConfig('Configuration')
+
+    #if config['IgnoreRuntimeWarnings']:
+    #    import warnings
+    #    warnings.filterwarnings(action="ignore", category=RuntimeWarning)
+
+
+    import inspect
+    import os.path
+
+    GangaRootPath = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))), '../..'))
+    def transform(x):
+        return os.path.normpath(Ganga.Utility.files.expandfilename(os.path.join(GangaRootPath,x)))
+
+    paths = map(transform, filter(None, map(lambda x: os.path.expandvars(os.path.expanduser(x)), config['RUNTIME_PATH'].split(':'))))
```

Comment:
it seems that
```python
config['RUNTIME_PATH']
```
is empty here even though it's set in my gangarc. I assume it hasn't read it in properly yet